Assignment2/sebufer-SergioBujanFernandez2.json

```diff
@@ -0,0 +1,31 @@
+"@context":{
+		"zar":"https://www.w3.org/api/general",
+		"rdfs":"https://www.w3.org/api/general#",
+		"name":"rdfs:label",
+		"hometown": "rdfs:label",
+		"age": "rdfs:integer",
+		"alma_mater": "rdfs:label",
+		"parents": "rdfs:label",
+		"subjects": "rdfs:subjects",
+		"name": "rdfs:label",
+		"teachers": "rdfs:label",
+		"description": "rdfs:label",

```

Comment:
Check my comments on attributes definition from those I provided for your other json file.